packages/terra-table/src/Table.jsx

```diff
@@ -0,0 +1,326 @@
+import React, {
+  useState, useContext, useRef, useCallback, useEffect, useMemo,
+} from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import classNames from 'classnames/bind';
+import ResizeObserver from 'resize-observer-polyfill';
+import ThemeContext from 'terra-theme-context';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
```

Comment:
Can we "group" terra imports separately?
```suggestion
import ResizeObserver from 'resize-observer-polyfill';

import ThemeContext from 'terra-theme-context';
import VisuallyHiddenText from 'terra-visually-hidden-text';
```

Reply:
Sorted and grouped imports in all files.